auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue

```diff
@@ -66,23 +66,32 @@ export default class DuplicateAccountWarningView extends Vue {
     @OrgModule.State('currentOrganization') private currentOrganization!: Organization
     @OrgModule.Action('addOrgSettings') private addOrgSettings!: (currentOrganization: Organization) => Promise<UserSettings>
     @OrgModule.Action('syncOrganization') private syncOrganization!: (orgId: number) => Promise<Organization>
+    @UserModule.Action('getUserAccountSettings') private getUserAccountSettings!: () => Promise<any>
     private orgsOfUser: OrgWithAddress[] = []
     private isLoading: boolean = false
     @Prop({ default: '' }) redirectToUrl !: string
 
     private async mounted () {
+      if (!Array.isArray(this.currentUserAccountSettings) || !this.currentUserAccountSettings.length) {
+        await this.getUserAccountSettings()
+      }
+    }
+    @Watch('currentUserAccountSettings', { immediate: true })
+    private onCurrentUserAccountSettings (): void {
       try {
-        this.isLoading = true
-        this.currentUserAccountSettings.map(async (accountsetting: UserSettings) => {
-          const orgId = parseInt(accountsetting.id)
-          const orgAdminContact = await this.getOrgAdminContact(orgId)
-          const orgOfUser: OrgWithAddress = {
-            id: orgId,
-            name: accountsetting.label,
-            addressLine: `${orgAdminContact.street} ${orgAdminContact.city} ${orgAdminContact.region} ${orgAdminContact.postalCode} ${orgAdminContact.country}`
-          }
-          this.orgsOfUser.push(orgOfUser)
-        })
+        if (Array.isArray(this.currentUserAccountSettings) && this.currentUserAccountSettings.length) {
+          this.isLoading = true
+          this.currentUserAccountSettings.map(async (accountsetting: UserSettings) => {
+            const orgId = parseInt(accountsetting.id)
+            const orgAdminContact = await this.getOrgAdminContact(orgId)
+            const orgOfUser: OrgWithAddress = {
+              id: orgId,
+              name: accountsetting.label,
+              addressLine: `${orgAdminContact.street} ${orgAdminContact.city} ${orgAdminContact.region} ${orgAdminContact.postalCode} ${orgAdminContact.country}`
+            }
+            this.orgsOfUser.push(orgOfUser)
```

Comment:
better to push all together after loop? to avoid rendering on each push

Reply:
Good call. I shall await till all the async operations are resolved